.github/workflows/release.yml

```diff
@@ -27,6 +27,7 @@ jobs:
     strategy:
         matrix:
             os: [ubuntu-latest, macos-latest]
+            python-version: ["3.10", "3.11", "3.12", "3.13"]
```

Comment:
add `, "3.x"` so we also get the latest, in case there's a new version out and we miss something.

Reply:
Fixed